File: serve.rb
Dir.chdir('/Users/balajiiswaroopandhavarapu/Downloads/CLAUDE')
require 'webrick'
server = WEBrick::HTTPServer.new(Port: 3456, DocumentRoot: '/Users/balajiiswaroopandhavarapu/Downloads/CLAUDE', AccessLog: [], Logger: WEBrick::Log.new('/dev/null'))
trap('INT') { server.shutdown }
trap('TERM') { server.shutdown }
server.start

File: serve.py
import os
os.chdir('/Users/balajiiswaroopandhavarapu/Downloads/CLAUDE')
import http.server, socketserver
PORT = 3456
Handler = http.server.SimpleHTTPRequestHandler
with socketserver.TCPServer(("", PORT), Handler) as httpd:
    httpd.serve_forever()
